cartography/intel/okta/users.py

```diff
@@ -43,11 +43,13 @@ def _get_okta_users(user_client):
 
 def transform_okta_user_list(okta_user_list):
     users = []
+    user_ids = []
 
     for current in okta_user_list:
         users.append(transform_okta_user(current))
+        user_ids.append(current.id)
```

Comment:
Each object of the `users` list already has an ID: https://github.com/lyft/cartography/blob/6954bc838dc5c2a3d20e0d0c4fcb02d21395b6aa/cartography/intel/okta/users.py#L72

Is this additional `user_ids` array needed?

Reply:
it avoid keeping the entire data of each users in memory for no reason. This array can be very large.